pkg/node/logfields.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package node
+
+const (
+	fieldRoute = "route"
```

Comment:
So, which package will have its logfields defined? CC @raybejjani

Reply:
log fields which are specific to a package don't need to be global but I'm open to a discussion

Reply:
Good question. `fieldRoute` is used in an Error message, and it is likely to be common in logs. I'd vote to put it in logfields.

Reply:
Moved to `logfields`